source/Crew.h

```diff
@@ -0,0 +1,68 @@
+/* Crew.h
+Copyright (c) 2019 by Luke Arndt
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef CREW_H_
+#define CREW_H_
+
+#include "Ship.h"
+
+class Crew
+{
+public:
+	// Calculate one day's salaries for the Player's fleet
+	static int64_t CalculateSalaries(
```

Comment:
Almost all of these class methods' arguments should be by const ref to avoid copies. Primitives can be just bool by value.
Generally the declarations are single-line, as well.

Reply:
Could you give me an example of how to do this? GitHub suggestions are really great for this. You can make a suggestion using `ctrl+g` (or command+g if you're on MacOS).

Reply:
`foo(const T t)` takes variable t of type `const T` by value, and must copy it.
`foo(T &t)` takes variable t of type `T` by non-const reference.
`foo(const T *t)` takes a pointer t by value that dereferences to an object of type `const T`

Reply:
Okay, I did some reading and I think I'm passing by const ref now.

Is `foo(const T &t)` what you're after?